LEAF_Request_Portal/admin/templates/mod_svcChief.tpl

```diff
@@ -342,6 +418,31 @@ function initiateModal(serviceID = 0, serviceName = '') {
     }
 }
 
+function pruneMember(groupID, userID) {
```

Comment:
Probably want to make sure these inputs are valid before we ping the db

Reply:
done